src/pljson.type.decl.sql

```diff
@@ -93,7 +93,9 @@ create or replace type pljson force under pljson_element (
 
   /* map functions */
   member function get_keys return pljson_list,
-  member function get_values return pljson_list
+  member function get_values return pljson_list,
+
+  overriding member function internal_path_put(self in out nocopy pljson, path pljson_path, elem pljson_element, path_position pls_integer) return boolean
```

Comment:
It's been a long time since I have written PL/SQL, so I might be wrong, but can't you leave this our of the declaration file and simply define it in the implementation to keep it truly internal? Defining it here makes it publicly accessible, right?

Reply:
Unfortunately this is not possible.